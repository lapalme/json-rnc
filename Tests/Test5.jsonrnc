## a comment to skip
# test of error recovery in the parse
start = [person]
tst = {a:b}
person = {name:string,
          id:(string|{no:number,w:string}),
          address:number@(maximum=100 exclusiveMinimum=10),
          postalCode? : [cpRE]
}
cpRE = /[A-Z][0-9][A-Z] [0-9][A-Z][0-9]/
b=[({w:null, c:integer})]
