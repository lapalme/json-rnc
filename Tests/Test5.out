1:[{'name': 'Guy', 'id': 'Lapalme', 'address': 45, 'postalCode': 'H0H 0H0'}, {'id': {'no'...e': None}]
[0]/(#/definitions/person)/postalCode	array expected:	H0H 0H0
{'no': 24} does not match any alternative:
 -[1]/id	string expected:	{'no': 24}
 -[1]/id	missing required field:w	
[2]/name	string expected:	null
true does not match any alternative:
 -[2]/id	string expected:	true
 -[2]/id	object expected:	true
[2]/address	illegal value:	3 <= 10 excl
1 objects read: 1 invalid, 0 bad, 0 with duplicate fields
Error Statistics
              1	[0]/(#/definitions/person)/postalCode:array expected:
              1	{'no': 24} does not match any alternative:
